installer/install.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/sh -e
+# command and configuration setup under XDG Base Directory Specification
+# https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+# @ NAME_ORIG       command name (original)
+# @ NAME_ALT        command name (alternative under XDG)
+# @ REPO_NAME       config var for git repo name (w/ or w/o "https://github.com/")
+# @ CLONE_NAME      config var for git repo checkout dir name
+cd >/dev/null || true
+HOME=$(pwd)
+help () {
+  echo "syntax: ${0##*/} NAME_ALT"
+  echo "syntax: ${0##*/} NAME_ALT REPO_NAME"
+  echo "syntax: ${0##*/} NAME_ORIG NAME_ALT REPO_NAME"
+  echo "syntax: ${0##*/} NAME_ORIG NAME_ALT REPO_NAME CLONE_NAME"
+  echo
+  echo "Set up a NAME_ALT command executed under the XDG based alternative"
+  echo "environment with its configuration from REPONAME for a NAME_ORIG"
+  echo "command."
+  echo
+  echo " * If NAME_ORIG is missing, 'nvim' is used."
+  echo " * If CLONE_NAME is missing, NAME_ORIG is used."
+  echo " * reqirement: NAME_ORIG !=  NAME_ALT"
+  echo
+  echo "'\$PATH' needs to include '~/bin'."
+  echo "REPO_NAME can use short form for GitHub."
+  echo "Default REPO_NAME is 'AstroNvim/AstroNvim'"
+}
+
+if [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  help
+  exit 0
+elif [ $# = 1 ]; then
+  NAME_ORIG="nvim"
+  NAME_ALT="$1"
+  REPO_NAME="AstroNvim/AstroNvim"
+  CLONE_NAME="$NAME_ORIG"
+elif [ $# = 2 ]; then
+  NAME_ORIG="nvim"
+  NAME_ALT="$1"
+  shift
+  REPO_NAME="$1"
+  shift
+  CLONE_NAME="$NAME_ORIG"
+elif [ $# = 3 ]; then
+  NAME_ORIG="$1"
+  shift
+  NAME_ALT="$1"
+  shift
+  REPO_NAME="$1"
+  shift
+  CLONE_NAME="$NAME_ORIG"
+elif [ $# = 4 ]; then
+  NAME_ORIG="$1"
+  shift
+  NAME_ALT="$1"
+  shift
+  REPO_NAME="$1"
+  shift
+  CLONE_NAME="$1"
+else
+  help
+  exit 0
+fi
+if [ "$NAME_ORIG" = "$NAME_ALT" ]; then
+  help
+  exit 1
+fi
+
+if [ -z "$REPO_NAME" ]; then
+  # undocumented: just create an alternative command without configuration
+  REPO_NAME=""
+else
+  if [ "$REPO_NAME" = "${REPO_NAME##http}" ]; then
+    REPO_NAME="https://github.com/$REPO_NAME"
+  fi
+  if [ "$REPO_NAME" = "${REPO_NAME%.git}" ]; then
+    REPO_NAME="$REPO_NAME.git"
+  fi
+fi
+
+# initialize
+BIN_DIR="$HOME/bin"  # may be ~/.local/bin
+XDG_CONFIG_HOME="$HOME/.config/$NAME_ALT"
```

Comment:
These should use parameter expansion to respect the `XDG_*` base directories if they have been set by the user to something other than the defaults., e.g.,

```suggestion
XDG_CONFIG_HOME="${XDG_CONFIG_HOME:-$HOME/.config}/$NAME_ALT"
```